ETL/Repository/DTO/AddEmployee.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ETL.Repository.DTO
+{
+    public class AddEmployee
```

Comment:
I thought classes were supposed to be nouns  :P

Reply:
Yeah, that one is weird because I'm trying to show that it correlates to that specific query.  I should probably make them more general